README.md

```diff
@@ -133,9 +143,46 @@ The response will be a stringified json object:
  timestamp: "<timestamp that you passed in or that was generated>"
 }
 ```
+
+You can also confirm the signature that you receive when you interacting with our servers, simply use the confirm_sec_key method which returns a boolean:
+
+```java
+import smile.identity.core.Signature;
+
+try {
+  Signature connection = new Signature(partner_id, api_key);
+  String signatureJsonStr = connection.confirm_sec_key(sec_key, timestamp);
+  // If it is valid then use the response, else throw an error
+} catch (Exception e) {
+  e.printStackTrace();
+  throw e;
+}
+```
+
+#### Utilities Class
+
+You may want to receive more information about a job. This is built into Web Api if you choose to set return_job_status as true in the options class. However, you also have the option to build the functionality yourself by using the Utilities class. Please note that if you are querying a job immediately after submitting it, you will need to poll it for the duration of the job.
+
+```java
+import smile.identity.core.Utilities;
+
+String job_status = new Utilities(<partner_id>, <the decoded-version of-your-api-key>, <sid_server>).get_job_status(<user_id>, <job_id>, <return_image_links> , <return_history>);
```

Comment:
Bill had a rather strenuous objection to the Utilities class idea on Friday. Before approving this structure we probably should discuss with him, as he was adamant that we were completely wrong on this concept for some reason.

Reply:
I've kept the Utilities class but used it to expose a method on the Web api class